helpers/postgresql_dump.sh

```diff
@@ -1,12 +1,34 @@
 #!/bin/bash
+# Script to create backup files for rhel6 to 7 migration.
+#
 # Notes:
-# 1. Use this script to create postgresql dumps - foreman.dump and candlepin.dump from the Satellite server.
-# 2. This scripts stops the katello-service in Satellite.  Please plan accordingly for the outage.
+# 1. This script runs katello-backup and also creates foreman.dump and candlepin.dump files from your Satellite6 server.
+# 2. Run this script as root user in Satellite6 server.
+# 3. This script stops the katello-service in Satellite.  Please plan accordingly for the outage.
+
+set -x
+
+# Create backup folder
 export backup_dir="/backup"
 mkdir $backup_dir
+
+# Run katello-backup
 katello-backup $backup_dir
+echo "pgsql_data.tar.gz is not required for rhel6 to 7 migration..."
+rm -f $backup_dir/katello-backup*/pgsql_data.tar.gz
```

Comment:
@sthirugn the only thing I can think of is to grab the folder name from the second line of output here:
```
Starting backup: 2017-03-29 19:20:12 +0000
Creating backup folder /backup/katello-backup-2017-03-29T19:20:12+00:00
``` 
but then there is the issue of the special characters...

I'll think about it some more though

Maybe we can just print a reminder to the user to not copy the psql_data.tar.gz file  over when doing a RHEL migration